build/__pythonVersions.sh

```diff
@@ -5,7 +5,7 @@ PIP_VERSION='21.2.4'
 PYTHON27_VERSION='2.7.18'
 PYTHON36_VERSION='3.6.15'
 PYTHON37_VERSION='3.7.15'
-PYTHON38_VERSION='3.8.16'
+PYTHON38_VERSION='3.8.15'
```

Comment:
Change after running `./build/generateConstants.sh` since the script was not ran during revert